### PR TITLE
Updates to the README file.

### PR DESCRIPTION
**Pull request by scottdangelo**
Some changes to the README for grammar and style.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ IBM Watson Natural Language Understanding API output through configurable mechan
...
 * Store the processed output JSON in DSX Object Storage.
 
-The intended audience for this journey is developers who want to learn method for augumenting classification metadata obtained from Watson Natural Language Understanding API, in situations when there is scarcity of historical data. The traditional approach of training a Text Analytics model yields less than expected results. The distinguishing factor of this journey is, it allows a configurable mechanism of text classification. It helps developer with a head start in case of text from specialized domain with no generally available english parser.
+The intended audience for this journey is developers who want to learn a method for augumenting classification metadata obtained from Watson Natural Language Understanding API, in situations when there is scarcity of historical data. The traditional approach of training a Text Analytics model yields less than expected results. The distinguishing factor of this journey is that it allows a configurable mechanism of text classification. It helps give a developer a head start in the case of text from a specialized domain,  with no generally available english parser.
```
Comment (markstur) [inline README.md]: s/scarcity/a scarcity/ (I think)
extra space after last comma
s/english/English/

File: README.md [hunk 2]
```diff
@@ -76,22 +78,24 @@ Click on `Add notebooks` (upper right) to create a notebook.
...
-Use `Find and Add Data` (look for the `10/01` icon)
-and its `Files` tab. From there you can click
-`browse` and add data and configuration files from your computer.
 
-> Note:  If you don't have your own data and configuration files, you can get our example by cloning
-this git repo. Look in the `data/sample_text.txt` directory for data file and in the `configuration/sample_config.txt`.
-If you use configuration file from your computer, make sure to conform to the JSON structure given in `configuration/sample_config.txt`.
+* From the `My Projects > Default` page, Use `Find and Add Data` (look for the `10/01` icon)
+and its `Files` tab. 
+* Click `browse` and navigate to this repo `watson-document-classifier/data/sample_text.txt
```
Comment (markstur) [inline README.md]: Looks like this line and next line are missing the ending back-tick around the file names

File: README.md [hunk 3]
```diff
@@ -163,11 +173,11 @@ Follow the link to open your Object Store Service in Bluemix.
 
 After running each cell of the notebook under Classify text, the results will display. 
 
-The configuration json controls the way the text is classfied. The classification process is divided into stages - Base Tagging and Domain Tagging. The Base Tagging stage can be used to specify keywords based classification, regular expression based classification and tagging based on chunking expressions. The Domain Tagging stage can be used to specify classification that are specific to the domain to augment the results from Watson Natural Language Understanding.
+The configuration json controls the way the text is classfied. The classification process is divided into stages - Base Tagging and Domain Tagging. The Base Tagging stage can be used to specify keywords based classification, regular expression based classification, and tagging based on chunking expressions. The Domain Tagging stage can be used to specify classification that is specific to the domain, in order to augment the results from Watson Natural Language Understanding.
```
Comment (markstur) [inline README.md]: s/classfied/classified/

File: README.md [hunk 4]
```diff
@@ -76,22 +78,24 @@ Click on `Add notebooks` (upper right) to create a notebook.
...
+* Click `browse` and navigate to this repo `watson-document-classifier/configuration/sample_config.txt
 
 ![](doc/source/images/add_file.png)
 
-#### Fix-up variable names
-Once the files have been uploaded into ``DSX-ObjectStore`` you need to update the variables that refer to the data and configuration files in the Jupyter Notebook.
+> Note:  It is possible to use your own data and configuration files.
+If you use a configuration file from your computer, make sure to conform to the JSON structure given in `configuration/sample_config.txt`.
+
+#### Fix-up variable names for your own data and configuration files
```
Comment (markstur) [inline README.md]: "Fix-up variable names" makes more sense with inserted code like credentials_2 --> credentials_1.  I think this notebook isn't using inserted code here so probably s/Fix-up variable names/Fix-up file names/ or something like that.

**Review by markstur — CHANGES_REQUESTED** [review]
see comments

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (scottdangelo) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (scottdangelo) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -54,14 +54,16 @@ Sign up for IBM's [Data Science Experience](http://datascience.ibm.com/). By sig
 
 ## 2. Create Bluemix services
 
-Create the following Bluemix service by following the link to use the Bluemix UI and create it.
+Create the following Bluemix service and name it wdc-NLU-service:
```
Comment (stevemar) [inline README.md:57]: could use backticks here: ``wdc-NLU-service``

**Review by stevemar — APPROVED** [review]
LGTM

**Comment by stevemar** [comment]
latest revision looks fine to me, thanks for the clean up @scottdangelo

**Pull request merged**